modules/controlfreec/1.1/config/freec/config_WGS.txt

```diff
@@ -13,22 +13,44 @@ gemMappabilityFile = referenceFile
 outputDir = OUTDIR
```

Comment:
Should there be a separate `config_WES.txt` file for exomes?

Reply:
Yes there will be. I need to think about the best way to implement exomes, since the default parameters and config file would be separate. I think switch_on_wildcards should be the easiest way to go at it. I'll likely get to the WES part in a future update since I need to test default WES parameters first (probably on the LLMPP exome data)

Reply:
Let's just document that this version is not recommended for exomes and make sure capture data isn't in the default config.